labs/src/main/scala/org/scalalabs/basic/lab03/RecursionPatternMatchingExercise.scala

```diff
@@ -28,16 +28,25 @@ object RecursionPatternMatchingExercise {
    * checkValuesIncrease(Seq(1,2,3)) == true
    * checkValuesIncrease(Seq(1,2,2)) == false
    */
-  def checkValuesIncrease(seq: Seq[Int]): Boolean = {
-    error("fix me")
-  }
+	def checkValuesIncrease(seq: Seq[Int]): Boolean = {
+			seq match {
+			case Nil => true
+			case a::Nil => true
+			case a::(b::rest) => if (a < b) checkValuesIncrease(b::rest) else false
+			}
+	}
```

Comment:
Strange. I think I accidentally used ScalaIDEs (command-I) to correct indentation, which defaults to (8?) spaces instead of two